galaxy/values-cvmfs.yaml

```diff
@@ -33,6 +33,40 @@ persistence:
   size: 10Gi
   mountPath: /galaxy/server/database
 
+#initContainers:
+  #myFirstContainer:
+    #applyToJob: true
+    #applyToWeb: true
+    #image: myrepo/myimage:mytag
+    #command: "['sh', '-c', 'cp my-path/to/my/file {{.Values.persistence.mountPath}}/custom-files']"
+    #volumeMounts:
+      #- name: galaxy-data
+        #mountPath: "{{.Values.persistence.mountPath}}"
+  #mySecondContainer:
+    #applyToJob: true
+    #applyToWeb: false
+    #image: myrepo/myimage:mytag
+    #command: "['sh', '-c', 'cp something {{.Values.persistence.mountPath}}/custom-files ']"
+    #volumeMounts:
+      #- name: galaxy-data
+        #mountPath: "{{.Values.persistence.mountPath}}"
+
+#initScripts:
```

Comment:
Should probably be extraInitScripts.

Reply:
Can change this too

Reply:
Although looking at it, it seems that other charts are using `initScripts` not `extraInitScripts` I think

Reply:
Kept at `initScripts` as seen more commonly imo, although open to changing it if you think it's better!